lib/rubycritic/source_control_systems/git.rb

```diff
@@ -52,11 +52,16 @@ def travel_to_head
       end
 
       def self.switch_branch(branch)
-        uncommitted_changes? ? abort('Uncommitted changes are present.') : `git checkout #{branch}`
+        dirty = !uncommitted_changes.blank?
+        abort("Uncommitted changes are present: #{uncommitted_changes}") if dirty
+
+        `git checkout #{branch}`
```

Comment:
While we're at it, can this be changed so that it also uses the `self.git` method? `git "checkout #{branch}"` 🙏